Player.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+
+
+namespace Card_Game
+{
+    public class Player 
+    {
```

Comment:
From Table class perspective it would be cool to have IsRoundWinner property in Player Class

Reply:
And maybe Name prop for the display purpose?